onchain/rollups/test/foundry/util/LibBytes.sol

```diff
@@ -0,0 +1,27 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+pragma solidity ^0.8.8;
+
+library LibBytes {
+    /// @notice Generate an 32-byte word from a byte array.
+    /// @param data The byte array
+    /// @return The 32-byte word
+    function toBytes32(bytes memory data) internal pure returns (bytes32) {
```

Comment:
I think this function name can be misleading. What do you think of the name `toHashBytes32`. Similarly, `toHashAddress` and `toHashUint256`.
Also, wouldn't `keccak256` be shorter than `LibBytes.toBytes32`? But I don't mind having either of them.

Reply:
Fair enough. Let me fix those issues!